l5kit/l5kit/evaluation/metrics.py

```diff
@@ -1,6 +1,8 @@
-from typing import Callable
```

Comment:
are we sure we want to pile all metrics here? should we maybe create a new dedicated package?

Reply:
My plan was to have all metrics here unless there is a significant semantic difference among them, but I haven't checked each one of them. Do you think we should split them into different modules ? For me it is ok, but if we can have all metrics together it would be simple to use and find them.